migrations/versions/1060_add_external_id_column.py

```diff
@@ -0,0 +1,77 @@
+"""Add external ID column
+
+Revision ID: 1050
```

Comment:
These should ideally match the ID numbers used by Alembic defined below :trollface: